src/app/strategies/page.tsx

```diff
@@ -11,7 +11,6 @@ import no_record from '../../assets/images/no_record.png';
 import '../styles/strategies.css';
 
 const strayegyList: React.FC = () => {
```

Comment:
The first letter of the component name needs to be capitalized.

Reply:
done